improver/cli/spot_extract.py

```diff
@@ -149,6 +154,8 @@ def process(
     from improver.spotdata.spot_extraction import SpotExtraction
     from improver.utilities.cube_extraction import extract_subcube
 
+    if realization_collapse:
+        cube = WeightAndBlend("realization", "linear", y0val=0.5, ynval=0.5,)(cube)
```

Comment:
Why are we always using this and not:
```
cube = collapsed(cube, "realization", iris.analysis.MEAN)
cube.remove_coord("realization")
```
?

Reply:
Changed.